### PR TITLE
fix: exhaustive-deps

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/components/SearchModal/CurrencySearch.tsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ export function CurrencySearch({
   const tokenComparator = useTokenComparator(false)
 
   // if they input an address, use it
-  const isAddressSearch = isAddress(debouncedQuery)
+  const isAddressSearch = useMemo(() => isAddress(debouncedQuery), [debouncedQuery])
```
Comment (viet-nv) [inline src/components/SearchModal/CurrencySearch.tsx:165]: `useMemo` is worthless here because `isAddressSearch` is already a primitive type.

Comment (nguyenhuudungz) [reply src/components/SearchModal/CurrencySearch.tsx:165]: Agree.

Comment (nguyenhuudungz) [reply src/components/SearchModal/CurrencySearch.tsx]: Fixed.

**Comment by nguyenhoaidanh** [comment]
sound good, thank you

**Pull request merged**